daemon/main.cpp

```diff
@@ -602,7 +602,7 @@ bool Daemon::setup_listen_fds()
 
     myaddr.sun_family = AF_UNIX;
 
-    mode_t old_umask = -1U;
+    mode_t old_umask = (mode_t)-1U;
```

Comment:
odn't use C style cases - I think you want static_cast here?

Reply:
yes, personally I jut hate the extra typing :)